src/wazuh_db/wdb_delta_event.c

```diff
@@ -94,12 +94,15 @@ bool wdb_upsert_dbsync(wdb_t * wdb, struct kv const * kv_value, cJSON * data) {
                 } else {
                     field_value = cJSON_GetObjectItem(data, field_name);
                     if (NULL == field_value) {
-                        if (column->value.is_pk) {
+                        if (column->value.is_pk && 0 != strncmp(kv_value->value, "sys_programs", 12)) {
                             has_error = true;
                         } else {
                             field_value = wdb_dbsync_get_field_default(&column->value);
```

Comment:
As far as I can see, the structure `static struct column_list const TABLE_PACKAGES[PACKAGES_FIELD_COUNT+1] 
` sets `""` as a default value instead of `" "`.

Reply:
The same happens in `wdb_package_insert()`, when we receive NULL, we insert `""`.

Reply:
As we agreed, the default value will be left as "".

Reply:
Thank you.